src/H5TS.c

```diff
@@ -68,25 +68,47 @@ static herr_t H5TS__mutex_unlock(H5TS_mutex_t *mutex, unsigned int *lock_count);
 /*****************************/
 
 /* Global variable definitions */
-#ifdef H5_HAVE_WIN_THREADS
-H5TS_once_t H5TS_first_init_g;
-#else  /* H5_HAVE_WIN_THREADS */
-H5TS_once_t H5TS_first_init_g = PTHREAD_ONCE_INIT;
-#endif /* H5_HAVE_WIN_THREADS */
+H5TS_once_t H5TS_first_init_g
+#ifndef H5_HAVE_WIN_THREADS
+    = PTHREAD_ONCE_INIT
+#endif
+    ;
 
 /* Thread-local keys, used by other interfaces */
-H5TS_key_t H5TS_errstk_key_g; /* Error stack */
+/* Error stack */
+H5TS_key_t H5TS_errstk_key_g
+#ifdef H5_HAVE_WIN_THREADS
+    = TLS_OUT_OF_INDEXES
+#endif
+    ;
+
 #ifdef H5_HAVE_CODESTACK
-H5TS_key_t H5TS_funcstk_key_g; /* Function stack */
-#endif                         /* H5_HAVE_CODESTACK */
-H5TS_key_t H5TS_apictx_key_g;  /* API context */
+/* Function stack */
+H5TS_key_t H5TS_funcstk_key_g
+#ifdef H5_HAVE_WIN_THREADS
+    = TLS_OUT_OF_INDEXES
+#endif
+    ;
+#endif /* H5_HAVE_CODESTACK */
```

Comment:
Definitely not a fan of this format. I'd just duplicate the variable declaration, as is done below.

Reply:
I missed that one - but with 4 instances it was way too busy and the formatter insisted on tabs making it look wierder.

Reply:
I think the problem I had was the original had some comments that just made it looked messy!
Updated with better.

Reply:
Is the static attribute okay? I think I accidentally used the H5TS_key_t H5TS_cancel_key_s section as a template.

Reply:
Never mind - the action check told me!

Reply:
okay format is weird - but that's what it wants.